### DISCUSSION TITLE
Welcome to amica Discussions!

### DISCUSSION THREAD

**Discussion by kasumi-1** [semperai/amica#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChiefBroseph** [comment]
Is there local tts synthesis with coqui's xttsv2?

**Reply by kasumi-1** [reply]
Yes, check out https://github.com/semperai/amica/blob/master/docs/local-setup.md

And configure https://github.com/semperai/basic-openai-api-wrapper/blob/master/app.py#L17 to use XTTS.

**Comment by beqa777** [comment]
Where can I get it .vrma files, such as the default "idle_loop.vrma" project provides, or is there a way to convert Unity animations to .vrma format?

**Reply by morphles** [reply]
@beqa777 its very hard to find them. I just created PR that allows to use mixamo animations (super great fun!) hopefully it will be merged by @kasumi-1 :)

**Reply by kasumi-1** [reply]
They're merged now. @beqa777 you can import any mixamo animation into the public/animations/.private directory and run `npm run generate:paths` - thanks @morphles

**Comment by morphles** [comment]
Now I'm a bit disappointed in how much I'm enjoying this thing, being long time ST fan. But voice integration is another level, probably mainly thanks to VAD. And I saw that other issue/comment about function calling via ollama. I'm considering to contributing significantly more. I'm considering adding sqlite db, chat history, char selections and what not. Unless of course there is no desire for such server side storage to be included @kasumi-1 ? , then might just do in on my fork :)

**Reply by kasumi-1** [reply]
@morphles sounds amazing. I would love that. Yeah I'm open to whatever features in repo as long as they're generally useful (and further the goal of making it easier for people to empathize with ai :) 

I started on character import and share pages. I was thinking for char selection page.. some whitelist of characters (so running locally you could just update this json to include own list), plus the last ones you have loaded (or star/bookmark or something). One thing missing from what I have for this so far is some sort of screenshot / picture of the character. 

For memory and so on - definitely, I imagine it would be a top level item in settings, where you can configure which memory backend you'd like, or to enable / disable it.

**Comment by yume-arasaki** [comment]
Hey @morphles I sent you a message on Linkedin.

**Comment by oxbaker** [comment]
Is the web version at https://amica.arbius.ai/ not working atm? I tried from my desktop and tablet and both give me "Failed to get chat response. Type Error: Failed to Fetch". Checked both this morning and now.

**Comment by yume-arasaki** [comment]
Is the problem still happening? I think the demo server just ran out of money : ) Should all be working shortly after top-up.

**Comment by DieterDev** [comment]
Is Amica still being developed? I would like more info, or how can i help?

**Reply by yume-arasaki** [reply]
Hey yes it is still being worked on, we have a couple of things we could get help on, one comes to mind:

https://github.com/semperai/amica/issues/146  (mapping animations to triggers)

characters page (new (prompt to save previous), load, save for characters locally (VRM, character details such as name and system prompt):

https://github.com/semperai/amica/pull/98

We're working on a few things internally (vr/ar , API), various mini features) - let me know if something interests you!

**Comment by nikita228gym** [comment]
Issue with local VRMA animations and expressions not loading in Desktop version (v0.2.1)

I am using the Amica desktop version (v0.2.1) on Windows 10, and I've encountered several issues with character animations and expressions:
Local VRMA files: I cannot find the correct directory to place .vrma files. I've tried creating public/animations/private in the installation folder and %AppData%, but the "Animation" dropdown menu remains empty. Drag-and-drop also doesn't work.
Model Expressions: My VRM model (10MB) shows no sliders in the "Face/Expressions" section, even though the console logs show that the LLM is sending expression: happy commands.
Console Errors: The developer console shows multiple "Unhandled rejection" and "Removing initializer" errors during model load.

and there are no folders in the "file location" so that I can add or remove animations, they do not appear, not in folders, not in Amica itself in the pop-up menu.
<img width="710" height="257" alt="image" src="https://github.com/user-attachments/assets/541251b6-76ea-4652-9a20-41cfffb367f2" />
<img width="864" height="220" alt="image" src="https://github.com/user-attachments/assets/6f46f765-400a-408b-8a52-789519be5bee" />
